Deployment/Release practices.md

```diff
@@ -121,16 +123,16 @@ Once you release your update into production, your app publishing status will be
 - **Removed** - The app is no longer available on Google Play or for existing users
 - **Update pending** - The update has been submitted and is being processed
 
-## Timed publishing
+## Managed publishing
```

Comment:
There are certain exclusion to managed publishing, which may be useful for people to know. These are:

```
Increasing an existing staged roll-out to 100%
Updating your app’s ”Release notes” section
Changes to device exclusion rules
Managing testers
Unpublishing your app
Changes to your app’s In-app products page
Price changes
```

Taken from the [official docs](https://support.google.com/googleplay/android-developer/answer/9859654?_ga=2.247535119.965012475.1608635570-1169082672.1531832630) 🙂